syntax = "proto3";

package tutorial;

message StartServiceRequest {
    string service_name = 1;
    string repo_url = 2;
    string version = 3;
    string id = 4;
    int32 sequence = 5;

    string source = 10;
    string target = 11;
}

